modules/ROOT/pages/visual-debugger-concept.adoc

```diff
@@ -3,11 +3,11 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio debugger enables you to run your application from Studio in debug mode, and to configure breakpoints to stop the execution of your application at a specific message processor. Then, using the *Mule Debugger* view, you can inspect the content of the message as it exists at that point in the flow, and evaluate a DataWeave expression against it.
+With Anypoint Studio debugger, you can run your application from Studio in debug mode and configure breakpoints to stop the execution of your application at a specific message processor. In the *Mule Debugger* view, inspect the content of the message as it exists at that point in the flow and evaluate a DataWeave expression against it.
 
-The debugger works with Enterprise Edition runtimes and does not connect to Community Edition runtimes.
+The debugger works with Enterprise Edition runtimes and doesn't connect to Mule Kernel runtimes.
```

Comment:
Kernel init cap correct?

Reply:
According to [this entry](https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#:~:text=or%20%22Mule%20runtimes%22-,Mule%20Kernel,-Replaces%20Mule%20Community) in the [Term Presentation Reference](https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference), yes.